### DISCUSSION TITLE
Infinite scrolling lists best practices

### DISCUSSION THREAD

**Discussion by kekeimiku** [fltk-rs/fltk-rs#1590] [Q&A]
I have an idea how to implement these things, but wondered if someone had already tackled this problem.

**Comment by MoAlyousef** [comment]
I personally haven't. As I understand, a Pack inside a Scroll should only call draws for pack elements which are supposedly visible, so you shouldn't get the overhead of calling draws for every pack child.
You might want to inquire at the fltk discussions as well:
https://github.com/fltk/fltk/discussions
